app/customer.py

```diff
@@ -0,0 +1,44 @@
+import dataclasses
+
+from typing import List, Union
+
+from app.car import Car
+from app.shop import Shop
+
+
+@dataclasses.dataclass
+class Customers:
+    name: str
+    product_cart: dict
+    location: list
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]